include/intersectables/meshTriangle.h

```diff
@@ -19,16 +19,10 @@ class MeshTriangle : public Triangle {
   MeshTriangle(int, Material*, const Vector3f&, const Vector3f&, const Vector3f&, const Vector3f&,
                const Vector3f&, const Vector3f&);
 
-  void log() const {
-    std::cout << "face-index: " << faceId << std::endl;
-    std::cout << "vertices: " << std::endl;
-    a.log();
-    b.log();
-    c.log();
-    std::cout << "normals: " << std::endl;
-    na.log();
-    nb.log();
-    nc.log();
-  }
+  Vector3f getNormalA() const { return na; }
```

Comment:
I guess we could omit these getter-method and simply expose `na`, `nb` and `nc` since there are immutable (marked by const).

Reply:
Whatever your preferred style is. Be aware that this is more than a getter: It makes a copy of na.